core/modules/catalog/store/category/index.ts

```diff
@@ -10,6 +10,11 @@ export const categoryModule: Module<CategoryState, RootState> = {
   state: {
     list: [],
     current: {},
+    pagination: {
```

Comment:
should be also in route path with filters

Reply:
URL deep linking is an optional feature, moreover having filters in URLs is just a kind of “view” where the Vue filters is kind of “model” for rendering the query string; we shouldn’t remove it